x-pack/plugins/ml/server/models/data_visualizer/data_visualizer.ts

```diff
@@ -731,6 +752,7 @@ export class DataVisualizer {
           filter: filterCriteria,
         },
       },
+      ...(runtimeMappings !== undefined ? { runtime_mappings: runtimeMappings } : {}),
```

Comment:
question: what if `runtimeMappings` is an empty object? shall we assign it in this case?

Reply:
Updated here [`b68e736` (#95252)](https://github.com/elastic/kibana/pull/95252/commits/b68e7366c3f40de28280ba378449f074039923b6) and [`861d28f` (#95252)](https://github.com/elastic/kibana/pull/95252/commits/861d28fd24de093b745d02be688adef7f54d4be0)